tests/access_test.go

```diff
@@ -215,7 +214,7 @@ var _ = Describe("[rfe_id:500][crit:high][arm64][vendor:cnv-qe@redhat.com][level
 	})
 
 	Describe("With default kubevirt service accounts", func() {
-		DescribeTable("should verify permissions on resources are correct for view, edit, and admin", func(group string, resource string, clusterWide bool, accessRights ...rights) {
+		DescribeTable("should verify permissions on resources are correct for view, edit, and admin", decorators.Conformance, func(group string, resource string, clusterWide bool, accessRights ...rights) {
```

Comment:
I am little bit torn here...

Reply:
Can you elaborate? :)

Reply:
I think different deployments might want to have RBAC set up differently. Also the RBAC wasn't or might still not be on by default on Kubernetes side.